apps/api/src/app/workflows-v2/usecases/build-payload-schema/build-payload-schema.usecase.ts

```diff
@@ -87,4 +73,30 @@ export class BuildPayloadSchema {
 
     return processedValue;
   }
+
+  private async buildVariablesSchema(variables: string[]) {
```

Comment:
i think we lost the case of

 if (!controlValues.length) {
    return this.buildVariablesSchema(sanitizedControlValues);
      return emptyJsonSchema();	
    }
    
    
   if i remember correctly this case was meant to cover the mini preview on step create, before we create the initial control values

Reply:
If I am not mistaken, this should be covered as it as the functions can all handle empty variables.